clients/src/test/java/org/apache/kafka/clients/consumer/KafkaConsumerTest.java

```diff
@@ -1600,9 +1605,8 @@ public void testCloseInterrupt() throws Exception {
     @Test
     public void testCloseShouldBeIdempotent() {
         KafkaConsumer<byte[], byte[]> consumer = newConsumer((String) null);
-        consumer.close();
-        consumer.close();
-        consumer.close();
```

Comment:
Is the third one not necessary?

Reply:
Yes, idempotency can be verified by two calls to close(), third one just adds to the total running time of this test.

Reply:
I have modified this test to verify using deterministic methods.